.github/workflows/block-extractor-scheduler.yml

```diff
@@ -0,0 +1,68 @@
+name: Block Extractor Workflow
+
+on:
+  schedule:
+    - cron: '0 */3 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  RUST_LOG: info
+
+jobs:
+  backup-blocks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        database: [postgres, bigquery]
```

Comment:
Are we using postgres in gcp too?

Reply:
No

Reply:
Are we not running the scheduler for postgres ? I thought we were running for both postgres and bigquery.